user/src/com/google/gwt/user/client/ui/TabBar.java

```diff
@@ -208,7 +208,7 @@ public TabBar() {
 
     panel.setVerticalAlignment(HasVerticalAlignment.ALIGN_BOTTOM);
 
-    HTML first = new HTML("&nbsp;", true), rest = new HTML("&nbsp;", true);
+    HTML first = new HTML("&#xA0;", true), rest = new HTML("&#xA0;", true); // '&#xA0;': ePub/XHTML friendly &nbsp;
```

Comment:
To fix this style issue, maybe move the comment to the previous line - might make sense for other lines too.

Reply:
Sure. Done consistently for all occurrences.